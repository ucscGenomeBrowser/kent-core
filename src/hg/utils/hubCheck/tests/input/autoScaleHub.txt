hub bigWigAutoScaleTest
shortLabel bigWig AutoScale Test
longLabel bigWig AutoScale Test
useOneFile on
email genome-www@soe.ucsc.edu

genome mm39

track compBad
compositeTrack on
type bigWig
shortLabel example of bad track
longLabel  example of bad track

    track badTrack1
    parent compBad
    shortLabel bad track 1
    longLabel bad track 1
    type bigWig
    maxHeightPixels 60
    autoScale group
    bigDataUrl https://genome-test.gi.ucsc.edu/~hiram/hubs/Plants/araTha1/bbi/araTha1.gc5Base.bw

    track badTrack2
    parent compBad
    shortLabel bad track 2
    longLabel bad track 2
    type bigWig
    maxHeightPixels 60
    autoScale group
    bigDataUrl https://genome-test.gi.ucsc.edu/~hiram/hubs/Plants/araTha1/bbi/araTha1.gc5Base.bw

track compGood
compositeTrack on
type bigWig
autoScale group
shortLabel example of good track
longLabel  example of good track

    track goodTrack1
    parent compGood
    shortLabel good track 1
    longLabel good track 1
    type bigWig
    maxHeightPixels 60
    bigDataUrl https://genome-test.gi.ucsc.edu/~hiram/hubs/Plants/araTha1/bbi/araTha1.gc5Base.bw

    track goodTrack2
    parent compGood
    shortLabel good track 2
    longLabel good track 2
    type bigWig
    maxHeightPixels 60
    bigDataUrl https://genome-test.gi.ucsc.edu/~hiram/hubs/Plants/araTha1/bbi/araTha1.gc5Base.bw
