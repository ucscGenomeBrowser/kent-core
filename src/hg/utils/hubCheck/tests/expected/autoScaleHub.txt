Found 9 problems:
track "badTrack1" has value "group" for autoScale setting, valid autoScale values for individual bigWig tracks are "off" or "on" only. If "badTrack1" is part of a bigWig composite track and you want to use the "group" setting, only declare "autoScale group" in the parent stanza
track "badTrack2" has value "group" for autoScale setting, valid autoScale values for individual bigWig tracks are "off" or "on" only. If "badTrack2" is part of a bigWig composite track and you want to use the "group" setting, only declare "autoScale group" in the parent stanza
warning: missing description page for track: 'badTrack1'
warning: missing description page for track: 'badTrack2'
warning: missing description page for track: 'compBad'
warning: missing description page for track: 'compGood'
warning: missing description page for track: 'goodTrack1'
warning: missing description page for track: 'goodTrack2'
warning: missing hub overview description page (descriptionUrl setting)
